.terraform-version

```diff
@@ -1 +1 @@
-1.2.8
+1.6.3
```

Comment:
same comments as for the PR https://github.com/StratusGrid/terraform-aws-limits-monitor/pull/5